geopandas/tests/test_doc.py

```diff
@@ -0,0 +1,91 @@
+from textwrap import dedent
```

Comment:
Can you call this file `test_decorators` to match the filename of the code source?

Reply:
Here I thought called it `test_doc_decorator.py` would be more clear. Because next step would gather all decorators in one.

Reply:
Assuming that we may want to add more decorators later, the file should accommodate more that `doc` decorator. Therefore it is better to keep its name more general (`test_decorator.py`).